code/game/objects/items/devices/scanners.dm

```diff
@@ -253,9 +253,23 @@ GENE SCANNER
 		var/mob/living/carbon/C = M
 		var/list/damaged = C.get_damaged_bodyparts(1,1)
 		if(length(damaged)>0 || oxy_loss>0 || tox_loss>0 || fire_loss>0)
-			to_chat(user, "<span class='info'>\tDamage: <span class='info'><font color='red'>Brute</font></span>-<font color='#FF8000'>Burn</font>-<font color='green'>Toxin</font>-<font color='blue'>Suffocation</font>\n\t\tSpecifics: <font color='red'>[brute_loss]</font>-<font color='#FF8000'>[fire_loss]</font>-<font color='green'>[tox_loss]</font>-<font color='blue'>[oxy_loss]</font></span>")
-			for(var/obj/item/bodypart/org in damaged)
-				to_chat(user, "\t\t<span class='info'>[capitalize(org.name)]: [(org.brute_dam > 0) ? "<font color='red'>[org.brute_dam]</font></span>" : "<font color='red'>0</font>"]-[(org.burn_dam > 0) ? "<font color='#FF8000'>[org.burn_dam]</font>" : "<font color='#FF8000'>0</font>"]")
+			var/dmgreportlist = "<pre><span class='info'>&#9;Damage:&#9;&#9;\
+							<font color='red'><b>Brute</font></span>&#9;\
+							<font color='orange'>Burn</font>&#9;\
+							<font color='green'>Toxin</font>&#9;\
+							<font color='purple'>Suffocation</b></font><br>\
+
+							&#9;&nbsp;&nbsp;&nbsp;&nbsp;<span class='info'>Overall:&#9;\
+							<font color='red'>[brute_loss]</font>&#9;\
+							<font color='orange'>[fire_loss]</font>&#9;\
+							<font color='green'>[tox_loss]</font>&#9;\
+							<font color='purple'>[oxy_loss]</font></span><br>"
+			for(var/obj/item/bodypart/org in damaged) //head, left arm, right arm, etc.

```

Comment:
typed list is no bueno here.

Reply:
Do you mean the comment? If so, I'll remove it, although I can't see why is no bueno.

Reply:
No. I mean that you put the typed variable as the loop index.
Basically that makes the loop run istype on all the content to check if it's an obj/item/bodypart which isn't needed in this case.
It's more performant to do
```
for(var/o in damaged)
	var/obj/item/bodypart/org = o
```

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#forvara-in-list-vs-forvari-in-1-to-listlen
Read this for more information on the topic